llama_stack/apis/scoring_functions/scoring_functions.py

```diff
@@ -19,20 +19,50 @@
 from pydantic import BaseModel, Field
 from typing_extensions import Annotated
 
-from llama_stack.apis.common.type_system import ParamType
 from llama_stack.apis.resource import Resource, ResourceType
 from llama_stack.schema_utils import json_schema_type, register_schema, webmethod
 
 
 # Perhaps more structure can be imposed on these functions. Maybe they could be associated
 # with standard metrics so they can be rolled up?
-@json_schema_type
 class ScoringFnParamsType(Enum):
```

Comment:
Note that different `ScoringFunctionType` could have the same `ScoringFnParamsType`. E.g.

`ScoringFnParamsType`=`regex_parser` is used for both:

- ScoringFunctionType=`regex_parser` --> mapping to code 
- ScoringFunctionType=`regex_parser_math_response` --> mapping to code

Reply:
Alternative in https://github.com/meta-llama/llama-stack/pull/1580#issuecomment-2716617105